python/lsst/daf/butler/registry/tests/_registry.py

```diff
@@ -52,16 +52,17 @@
     ddl,
     Timespan,
 )
-from .._registry import (
-    CollectionSummary,
-    CollectionType,
+from .._sqlRegistry import Registry
```

Comment:
Should it be `from .._registry import Registry` or `from .._sqlRegistry import SqlRegistry`?

Reply:
Yes, that's funny.

Reply:
It works because it's only used for type annotation.